common/confparser.h

```diff
@@ -67,19 +68,20 @@ const char* parse_conf_string(const char* cfgfile, const char* expr,
 
 /** Common */
 const char* parse_conf_policy_filename(const char* cfgfile);
-const char* parse_conf_zonelist_filename(const char* cfgfile);
+const char* parse_conf_zonelist_filename_enforcer(const char* cfgfile);
+const char* parse_conf_zonelist_filename_signer(const char* cfgfile);
 const char* parse_conf_zonefetch_filename(const char* cfgfile);
 const char* parse_conf_log_filename(const char* cfgfile);
 
-/** Enforcer specific */
-const char* parse_conf_pid_filename(const char* cfgfile);
+/** Enforcer and Signer specific */
+const char* parse_conf_pid_filename(const char* cfgfile, int enfsig);
 const char* parse_conf_delegation_signer_submit_command(const char* cfgfile);
 const char* parse_conf_delegation_signer_retract_command(const char* cfgfile);
-const char* parse_conf_clisock_filename(const char* cfgfile);
-const char* parse_conf_working_dir(const char* cfgfile);
-const char* parse_conf_username(const char* cfgfile);
-const char* parse_conf_group(const char* cfgfile);
-const char* parse_conf_chroot(const char* cfgfile);
+const char* parse_conf_clisock_filename(const char* cfgfile, int enfsig);
```

Comment:
For me it is quite unclear form the header files what I would need to pass here as parameter for enfsig.  Sure I get that enfsig references a switch between enforcer and signer, but it is an int so I'd pass 0, or 1?  Or something else and would a 0 be enforcer or signer?
Either use as parameter name "isEnforcer" or "is_enforcer" so I would expect to pass 1 if it concerns the enforcer, or use two #define's with ENFORCER and SIGNER or use an enum {ENFORCER, SIGNER}.